src/index.html

```diff
@@ -6,17 +6,345 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+
+    <link
+      rel="shortcut icon"
+      href="./assets/favicon.ico"
+      type="image/x-icon"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Bike</title>
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./styles/index.scss"
     />
     <script
       src="scripts/main.js"
       defer
     ></script>
   </head>
   <body>
-    <h1>Landing Page</h1>
+    <header class="header">
+      <div class="container-header">
+        <img
+          src="./assets/logo-light.svg"
+          alt="logo My Bike"
+          loading="eager"
+        />
+
+        <div class="menu">
+          <a href="tel:+12345555555">
+            <img
+              src="./assets/icon-phone-call.png"
+              alt="ícone de telefone"
+              loading="eager"
+            />
+          </a>
+
+          <input
+            type="checkbox"
+            id="menu-toggle"
+            class="menu-toggle"
+          />
+          <label
+            for="menu-toggle"
+            class="menu-icon"
```

Comment:
Inconsistent usage of naming conventions:

The code sometimes uses a block-element format (e.g., menu-toggle) instead of the BEM-recommended block__element.

Example:
menu-icon should ideally be menu__icon.